docs/plugins.md

```diff
@@ -290,145 +290,12 @@ exports.addWarning(message, options);
 If you specify any of these properties in your plugin file, they will be
 overwritten.
 
```

Comment:
The docs above this need to be updated as well. they mention build in plugins, and the 'in this document' table of contents is wrong now.